### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by xiangxn**
update dependencies

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,15465 @@
...
+    },
+    "node_modules/alphanum-sort": {
+      "version": "1.0.2",
+      "resolved": "https://registry.npmjs.org/alphanum-sort/-/alphanum-sort-1.0.2.tgz",
+      "integrity": "sha1-l6ERlkmyEa0zaR2fn0hqjsn74KM=",
+      "dev": true
+    },
+    "node_modules/amdefine": {
+      "version": "1.0.1",
+      "resolved": "https://registry.npm.taobao.org/amdefine/download/amdefine-1.0.1.tgz",
```
Comment (abitmore) [inline package-lock.json:1812]: Can we stick with registry.npmjs.org?

File: package-lock.json [hunk 2]
```diff
@@ -1,8 +1,15465 @@
...
+    },
+    "node_modules/@types/node": {
+      "version": "14.0.27",
+      "resolved": "https://registry.npmjs.org/@types/node/-/node-14.0.27.tgz",
+      "integrity": "sha512-kVrqXhbclHNHGu9ztnAwSncIgJv/FaxmzXJvGXNdcCpV1b8u1/Mi6z6m0vwy0LzKeXFTPLH0NzwmoJ3fNCIq0g==",
+      "dev": true
+    },
+    "node_modules/@types/normalize-package-data": {
+      "version": "2.4.1",
+      "resolved": "https://registry.npmmirror.com/@types/normalize-package-data/download/@types/normalize-package-data-2.4.1.tgz?cache=0&sync_timestamp=1637269900376&other_urls=https%3A%2F%2Fregistry.npmmirror.com%2F%40types%2Fnormalize-package-data%2Fdownload%2F%40types%2Fnormalize-package-data-2.4.1.tgz",
```
Comment (abitmore) [inline package-lock.json:1519]: Can we stick with registry.npmjs.org?

**Review by abitmore — CHANGES_REQUESTED** [review]
Thanks.

**Comment by xiangxn** [comment]
Replaced `npm` with `yarn`, and used the source of yarn

**Comment by xiangxn** [comment]
Can this `PR` be merged?

**Comment by sschiessl-bcp** [comment]
Sorry, overlooked this one.

**Pull request merged**